moto/core/botocore_stubber.py

```diff
@@ -33,12 +33,17 @@ def reset(self) -> None:
         self.methods.clear()
 
     def register_response(
-        self, method: str, pattern: Pattern[str], response: Callable[..., TYPE_RESPONSE]
```

Comment:
There doesn't seem to be any functional changes here - can we keep the existing formatting? Just to try and keep the diff as small as possible

Reply:
I think I made a change here and then undid it, and the formatting was altered by accidental. Fixed.